src/Collections/Sequences/Sorting.dfy

```diff
@@ -0,0 +1,191 @@
+// RUN: %dafny /compile:0 "%s"
+
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT 
+*******************************************************************************/
+
+include "../../StandardLibrary.dfy"
+include "../../UInt.dfy"
+include "../Sequences/Seq.dfy"
+
+module Sorting {
```

Comment:
Could you use the attribute `{:options "/functionSyntax:4"}` and change all your `predicate method` to `predicate` ?

Reply:
This again